src/classes/SRVC_RelAutoCreateSettingsMapper.cls

```diff
@@ -0,0 +1,107 @@
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2020
+* @group 
+* @group-content ../../ApexDocContent/
+* @description Service class for REL_Relationships_Con_TDTM; used to generate MODL_ConRelationshipAutoCreateMapping
+* models representing Relationship Auto Create mappings set for the Contact object. 
+*/
+public virtual with sharing class SRVC_RelAutoCreateSettingsMapper {
+    /***************************************************************************************************************************
+    * @description Instance for Singleton Pattern.
+    ***************************************************************************************************************************/
+    @TestVisible
+    private static SRVC_RelAutoCreateSettingsMapper instance;
+
+    /***************************************************************************************************************************
+    * @description Empty constructor is not public for Singleton Pattern.
+    ***************************************************************************************************************************/
+    protected SRVC_RelAutoCreateSettingsMapper() {}
+
+    /***************************************************************************************************************************
+    * @description Static method to get the current instance for Singleton pattern.
+    * @return The instance of SRVC_RelAutoCreateSettingsMapper.
+    ***************************************************************************************************************************/
+    public static SRVC_RelAutoCreateSettingsMapper getInstance() {
+        if (SRVC_RelAutoCreateSettingsMapper.instance == null) {
+            SRVC_RelAutoCreateSettingsMapper.instance = new SRVC_RelAutoCreateSettingsMapper();
+        }
+
+        return SRVC_RelAutoCreateSettingsMapper.instance;
+    }
+
+    /***************************************************************************************************************************
+    * @description Retrieves a list of MODL_ConRelationshipAutoCreateMapping representing the Relationship Auto Create
+    * mappings set for the Contact object.
+    * @return A list of ContactRelationshipAutoCreateMapping models.
+    **************************************************************************************************************************/
+    public virtual List<MODL_ConRelationshipAutoCreateMapping> getContactMappingModelsFromSettings(){
+        Map<Schema.SObjectType, List<Relationship_Auto_Create__c>> relAutoCreateListBySObjectType = UTIL_CustomSettingsFacade.getAutoCreateSettingsForContact();
```

Comment:
Let's add a standard lazy load caching here for a static List<MODL_ConRelationshipAutoCreateMapping>. It wasn't in the original technical spec, but the potential of running multiple schema inspections should be cut off at the tap here.